main/ertp/api/purse.md

```diff
@@ -36,7 +36,40 @@ quatloosPurse1 = quatloosIssuer.makeEmptyPurse();
 quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
 ```
 
-## purse.getCurrentAmount()
+## purse.getRecentAmount()
+- Returns: `{Amount}`
+
+Get an `amount` describing the current digital assets balance in the `purse`.
```

Comment:
```suggestion
Get a promise for the `amount` of digital assets in the `purse`. Because the promise will resolve at a later point, this information may be outdated by the time it is received, but it was the latest and most accurate balance at the time that the method was called.
```